src/util/webhooks.ts

```diff
@@ -3,7 +3,17 @@ import { metricWebhooks } from "../config/metric-names";
 
 export const getCurrentTime = () => Date.now();
 
-export const emitWebhookProcessingTimeMetrics = (
```

Comment:
I've renamed the method hence I've added a counter metric here which counts how many webhooks were processed successfully